res/values/strings.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
 
-    <string name="app_name">Adblock Plus for Samsung Internet</string>
+    <string name="app_name">Adblock Plus</string>
```

Comment:
We still want to keep the current naming for the MainPreferences activity. Given that, instead of modifying the app_name string, we need to add a new item `<string name="launcher_name">Adblock Plus</string>` and use it in the application label on AndroidManifest.xml